src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionContext.java

```diff
@@ -96,31 +96,44 @@ protected ImmutableMap<String, String> getRemoteExecProperties() throws EvalExce
       name = "modules",
       structField = true,
       doc =
-          "A list of all the Bazel modules in the external dependency graph, each of which is a <a"
-              + " href=\"../builtins/bazel_module.html\">bazel_module</a> object that exposes all"
-              + " the tags it specified for this module extension. The iteration order of this"
-              + " dictionary is guaranteed to be the same as breadth-first search starting from the"
-              + " root module.")
+          "A list of all the Bazel modules in the external dependency graph that use this module "
+              + "extension, each of which is a <a href=\"../builtins/bazel_module.html\">"
+              + "bazel_module</a> object that exposes all the tags it specified for this extension."
+              + " The iteration order of this dictionary is guaranteed to be the same as"
+              + " breadth-first search starting from the root module.")
   public StarlarkList<StarlarkBazelModule> getModules() {
     return modules;
   }
 
   @StarlarkMethod(
       name = "is_dev_dependency",
```

Comment:
I think I'd prefer a separate method for this. Since its usage is so narrow, maybe even just `module_ctx.root_module_has_non_dev_dependency`.

Reply:
I like that, changed.